README.md

```diff
@@ -1,85 +1,60 @@
-# Morello UI
-A GUI for executing binaries on morello board
+# morello-ui
 
+A GUI for executing binaries on [Morello](https://www.arm.com/architecture/cpu/morello) and `aarch64` architecture. Binaries are executed using [`morello-api`](https://github.com/digicatapult/morello-api/).
 
-## Integration E2E testing
-This repository uses Cypress for testing UI. Specs can be found in `cypress/integration`.
-**Test Types**
-- `smoke` - checks if renders root component
+## Environment Variables
 
-### Local development
-Pretty much a standard one as for every other ReactJS app. There will be some .env that will require to be added/updated, but for time being it's to early.
+`morello-ui` is configured primarily using environment variables as follows:
```

Comment:
This isn't true. `morello-ui` can be configured at build time using these environment variables. To configure at runtime `config.js` should be replaced with a version that sets appropriate values for that environment. You may wish to split this explanation between here and the production section

Reply:
how about this:
```
Environment variables can be configured at **build** time using a `.env` file at the root of the repository.
Environment variables can be configured at **runtime** by editing [`config.js`](./public/config.js).
```

Reply:
Ignore above, I split it up further into
```
At **BUILD** time, `morello-ui` is configured using environment variables in a `.env` file at the root of the repository:

| variable      | required |   default   | description           |
| :------------ | :------: | :---------: | :-------------------- |
| DSBD_API_HOST |    N     | `localhost` | Morello API host name |
| DSBD_API_PORT |    N     |   `3001`    | Morello API port      |

At **RUNTIME**, edit [`config.js`](./public/config.js). Default values:

{
  dsbdApiPort: 3001,
  dsbdApiHost: 'localhost',
}
```